l10n_fi_banks/data/res_bank.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+
+        <!-- Finnish banks -->
+
+        <!-- TODO: Add main branch address information -->
+        <!-- TODO: Add branches? -->
+
+        <!-- Disable the Reserve-account -->
+        <!-- We could also delete this, but there is always a chance
+          someone modified this to be an actual account -->
+        <record id="base.res_bank_1" model="res.bank">
+            <field name="active" eval="0" />
+        </record>
+
+        <!-- Please maintain the alphabetical order when doing additions -->
```

Comment:
Consider adding the following banks as well.

- DnB NOR Bank ([DNBAFIHX](https://www.dnb.no/en/about-us/global-network/finland.html))
- Holvi ([HOLVFIHH](https://support.holvi.com/hc/en-gb/articles/201434362-Payments-to-Holvi-IBAN-and-BIC))